web/templates/draft.hbs

```diff
@@ -2,12 +2,14 @@
     <div class="draft-info-box" tabindex="0">
         {{#if is_stream}}
         <div class="message_header message_header_stream">
-            <div class="message-header-contents">
-                <div class="message_label_clickable stream_label {{dark_background}}"
-                  style="background: {{stream_color}}; border-left-color: {{stream_color}};">
+            <div class="message-header-contents" style="background: {{recipient_bar_color}};">
+                <div class="message_label_clickable stream_label">
+                    <span class="stream-privacy filter-icon"  style="color: {{stream_privacy_icon_color}}">
```

Comment:
We might as well let these colors live-update, right?

Reply:
Yes, updated!